lib/matplotlib/axis.py

```diff
@@ -306,6 +306,16 @@ def set_label2(self, s):
         self.label2.set_text(s)
         self.stale = True
 
+    def set_url(self, s):
+        """
+        Set the url of label1 and label2
+
+        ACCEPTS: str
+        """
```

Comment:
Can you also call `super(self, Tick).set_label(s)`?

Reply:
Hi Thomas, if you meant to add a call to `super(Tick, self).set_url(s)`, I just added it, if not, I don't get it, sorry.

Reply:
Travis is failing with that addition with some `ImageComparisonFailure: images not close` errors. When calling the parent Artist's set_url() method it just does `self._url = url`! Amazing.

Reply:
That is what I meant, thanks.

Can you reproduce the failures locally?